contracts/cw4-group/src/contract.rs

```diff
@@ -185,15 +184,28 @@ const DEFAULT_LIMIT: u32 = 10;
 
 fn list_members(
     deps: Deps,
-    start_after: Option<String>,
+    start_after: Option<(u64, String)>,
     limit: Option<u32>,
 ) -> StdResult<MemberListResponse> {
     let limit = limit.unwrap_or(DEFAULT_LIMIT).min(MAX_LIMIT) as usize;
-    let addr = maybe_addr(deps.api, start_after)?;
-    let start = addr.map(|addr| Bound::exclusive(addr.to_string()));
+    // FIXME: Avoid building the (length-prefixed) bound manually
+    let start = start_after.map(|(w, a)| {
+        Bound::exclusive(
+            [
+                b"\x00".as_ref(),
+                b"\x08".as_ref(),
+                &w.to_be_bytes(),
+                a.as_bytes(),
+            ]
+            .concat(),
+        )
+    });
 
-    let members: StdResult<Vec<_>> = MEMBERS
-        .range(deps.storage, start, None, Order::Ascending)
+    let members: StdResult<Vec<_>> = members()
+        .idx
+        .weight
+        .sub_prefix(()) // FIXME: Use range() directly, and impl EmptyPrefix for IntKey<T>?
```

Comment:
Removing this line, we get:
```
error[E0599]: the method `range` exists for struct `MultiIndex<'_, (cw_storage_plus::keys::IntKey<u64>, PkOwned), u64>`, but its trait bounds were not satisfied
   --> contracts/cw4-group/src/contract.rs:207:10
    |
207 |         .range(deps.storage, start, None, Order::Descending)
    |          ^^^^^ method cannot be called on `MultiIndex<'_, (cw_storage_plus::keys::IntKey<u64>, PkOwned), u64>` due to unsatisfied trait bounds
    | 
   ::: /usr2/home/mauro/work/cosmwasm-plus/packages/storage-plus/src/keys.rs:193:1
    |
193 | pub struct IntKey<T: Endian> {
    | ---------------------------- doesn't satisfy `_: cw_storage_plus::keys::EmptyPrefix`
    |
    = note: the following trait bounds were not satisfied:
            `cw_storage_plus::keys::IntKey<u64>: cw_storage_plus::keys::EmptyPrefix`

```

Reply:
This is because a `Prefix` for a tuple is of course not an `EmptyPrefix`. The solution straightforward: use `SubPrefix: EmptyPrefix` as marker type instead.

Will do the same for the other top-level `range()`s.